### PR TITLE
CALS-5565 RFA - Page Header - 01A, B, C

### PR DESCRIPTION
**Pull request by nasar-cambria**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-5565

#### Brief feature description
- Accept to see page header for RFA 01A. Page header should be visible when visiting RFA 01 A, B, C

#### Does this PR block/effect other stories?
- No

#### Do any commands needs to be run in order to run the project with this PR?
- No

#### Code coverage percent diff as a result of this PR?
- 82.97% Statements
  70.32% Branches
  78.8% Functions
  83.56% Lines

### CODE REVIEW

**Comment by nasar-cambria** [comment]
@lauherkcambria it's working fine for me. Did you rebuild-assets ?

**Comment by ratneshraval** [comment]
Sticky for me in Chrome but doesn't work in IE11 
screen shot 2018-03-15 at 9 05 43 am

File: app/javascript/rfa_forms/rfa_sidebar/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default class RfaSideBar extends React.Component {
 
       <div className='col-sm-12'>
         <div className='nav-menu col-sm-10'>
-          <Affix>
+          <Affix affixStyle={{top: 80}}>
```
Comment (ratneshraval) [inline app/javascript/rfa_forms/rfa_sidebar/index.js]: What does this do?

**Comment by nasar-cambria** [comment]
jenkins re-test this please

**Pull request merged**